generators/client/templates/react/src/main/webapp/app/shared/error/error-boundary.tsx.ejs

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+interface IErrorBoundaryProps {
+  readonly children: JSX.Element;
```

Comment:
I think it can be `JSX.Element[] `as well so may be `JSX.Element | JSX.Element[]` might be better